src/core/reducers/search.js

```diff
@@ -30,6 +37,12 @@ export default function search(state = initialState, action) {
           createInternalAddon(payload.entities.addons[slug])
         )),
       };
+    case ABORT_SEARCH:
+      return {
+        ...state,
```

Comment:
I think this also needs `count: 0`. I wonder why `SEARCH_STARTED` doesn't do that though. Hmm.